R/guides-.R

```diff
@@ -541,10 +541,10 @@ Guides <- ggproto(
     # prepare output
     for (i in vec_seq_along(groups)) {
       adjust <- NULL
-      position <- groups$key$position[i]
+      position <- groups$key$positions[i]
       if (position == "inside") {
         adjust <- theme(
-          legend.position.inside = groups$key$coord[[i]],
+          legend.position.inside = groups$key$coords[[i]],
```

Comment:
Did you by change go through the code to check if there were more of these partial matches?

Reply:
I'll add it to my todo list